modules/cugraph/src/renumber.ts

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) 2021, NVIDIA CORPORATION.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {DataFrame, Int32, Series, StringSeries, Utf8String} from '@rapidsai/cudf';
+
+function join(edges: DataFrame, nodes: DataFrame<{id: Int32, node: Utf8String}>, col: string) {
+  const tmp = edges.assign({node: edges.get(col)}).join({other: nodes, on: ['node']}).sortValues({
+    idx: {ascending: true}
+  });
+  return edges.drop([col]).assign({[col]: tmp.get('id')});
+}
+
+export function renumber_nodes(src: StringSeries, dst: StringSeries) {
```

Comment:
Any particular reason these are string series? I believe src/dst could be any dtype.

Reply:
Currently I believe there is one needed method that is only on string series (on phone atm will have to dig it up later) was going to make a separate issue top generalize

Reply:
Well, I can't find what I was thinking of before I will give a shot at generalizing it

Reply:
see https://github.com/rapidsai/node/pull/323/commits/e541219c493e060e5103cba79a58a2a1be3a5266